src/Apps/W1/Shopify/App/src/GraphQL/Codeunits/ShpfyGQLAcceptFFRequest.Codeunit.al

```diff
@@ -0,0 +1,29 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Integration.Shopify;
+
+codeunit 30412 "Shpfy GQL AcceptFFRequest" implements "Shpfy IGraphQL"
```

Comment:
Doc for the codeunit would be great. AcceptFFRequest? Accept Fulfillment Request? Accept Friends Family Request? :D